apics/static/Atlas/109.html

```diff
@@ -43,7 +44,7 @@
     (2) trisyllabic
 </div>
 <table>
-<tr><td>pikanin(i)</td> <td><a href="/contributions/61"><span>(Fanakalo; Mesthrie 2013)</span></a></td></tr>
+<tr><<td style="width: 60px">pikanin(i)</td> <td style="width: 60px"><a href="/contributions/61"><span>(Fanakalo; Mesthrie 2013)</span></a></td></tr>
```

Comment:
This `<<td` is surely unwanted, right?

Reply:
jap, true. an eye of an eagle. thanks a lot